eurovision/notebooks/story.ipynb

```diff
@@ -910,6 +928,14 @@
     "plt.savefig('../../plots/uk_scores.pdf', bbox_inches='tight')\n"
    ]
   },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compared to the UK, how do the _most successful_ countries perform over the years? We'll have a look at the top four countries, for both the total and the average scores."
```

Comment:
As an aside (since this doesn't relate to your changes) reading through this and your comments above makes me wonder whether Yugoslavia should be ommitted from the graph here?